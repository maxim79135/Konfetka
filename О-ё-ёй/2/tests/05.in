40
-96853 -16326 -57824 99737 43147 -81777 2710 61743 -72061 4664 -89524 -829 -50149 33374 -41334 -92102 14584 -63996 68713 24851 -44125 76760 80659 -22779 53726 -60135 -5966 54839 -25158 70647 45513 39344 -6165 -92044 6519 95510 -25064 -61563 11615 -75683 
