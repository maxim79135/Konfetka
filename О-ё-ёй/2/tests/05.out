99737 95510 80659 76760 70647 68713 61743 54839 53726 45513 43147 39344 33374 24851 14584 11615 6519 4664 2710 -829 -5966 -6165 -16326 -22779 -25064 -25158 -41334 -44125 -50149 -57824 -60135 -61563 -63996 -72061 -75683 -81777 -89524 -92044 -92102 -96853 